gnocchi/rest/api.py

```diff
@@ -157,7 +159,10 @@ def deserialize(expected_content_types=None):
     try:
         params = json.load(pecan.request.body_file)
     except Exception as e:
-        abort(400, "Unable to decode body: " + six.text_type(e))
+        details = "Unable to decode body: %s \n" \
+                  "Body: %s" % (six.text_type(e), pecan.request.body_file)
+        LOG.warning(details)
```

Comment:
just curious, is there a reason for double logging at different levels?

Reply:
I talked with Pedro about that. The log in debug level of the `abort` command will affect other errors as well. However, this specific one, in our opinion, should generate a warning message as well. So, when it happens, the operators can see the error and the request body that generated it.